### PR TITLE
esm.sh is throwing a 500 error for this import

### PR DESCRIPTION
**Pull request by justinmchase**
error: Import 'https://cdn.esm.sh/extract-files' failed: 500 Internal Server Error
    at https://deno.land/x/graphql_request@deno-3.7.0/src/createRequestBody.ts:2:66

### CODE REVIEW

File: src/createRequestBody.ts [hunk 1]
```diff
@@ -52,14 +52,14 @@ export default function createRequestBody(
...
+  files.forEach((paths: string[]) => {
     map[++i] = paths
   })
   form.append('map', JSON.stringify(map))
 
   i = 0
-  files.forEach((paths, file) => {
-    form.append(`${++i}`, file as any)
+  files.forEach((_paths: string[], file: string | Blob) => {
+    form.append(`${++i}`, file)
```
Comment (justinmchase) [inline src/createRequestBody.ts:62]: The latest version of Deno doesn't like these `any` types, typing them more strictly fixes some runtime errors also.

File: src/createRequestBody.ts [hunk 2]
```diff
@@ -1,6 +1,8 @@
 // deno-lint-ignore-file no-explicit-any
-import { isExtractableFile, extractFiles, ExtractableFile } from 'https://cdn.esm.sh/extract-files'
+import * as ext from 'https://cdn.esm.sh/extract-files@12.0.0/extractFiles.mjs'
+import { ExtractableFile } from 'https://cdn.esm.sh/extract-files@12.0.0/extractFiles.mjs'
```
Comment (justinmchase) [inline src/createRequestBody.ts:3]: The type is exported as expected but the functions are exported slightly differently.

**Comment by isanasan** [comment]
@JumpLink

I want you to merge the PR .

**Comment by JumpLink** [comment]
@isanasan Thanks I'll take a look

**Comment by justinmchase** [comment]
No problem, it wasn't blocking me. Just something I noticed while messing with it.

**Pull request merged**